docs/archived/locked-inc-ftp/figures/firmware/source/src/main.c

```
@@ -88,23 +93,19 @@ static esp_err_t initialize_hardware_buses(star_bus_manager_t* const bus_manager
-                                           .output_mode   = k_pca9685_output_totem_pole,
-                                           .ext_clock     = false,
-                                           .invert_output = false};
-
-  esp_err_t ret = star_sensor_pca9685_init(pca_handle,
-                                           bus_manager,
-                                           STAR_SYSTEM_CONFIG.bus_names.pca9685,
-                                           error_iface,
-                                           STAR_SYSTEM_CONFIG.gpio.pca9685_oe,
-                                           false,
-                                           &pca9685_config);
+static esp_err_t initialize_pca9685(star_bus_manager_t *const bus_manager,
```

Clang (14.0.6)

[warning] 3 adjacent parameters of 'initialize_pca9685' of similar type ('int *const') are easily swapped by mistake

(bugprone-easily-swappable-parameters)